doc/api/next_api_changes/removals/24254-OG.rst

```diff
@@ -0,0 +1,64 @@
+Removal of deprecations in the Selector widget API
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+RectangleSelector and EllipseSelector
+.....................................
+
+The *drawtype* keyword argument to `~matplotlib.widgets.RectangleSelector` is
+removed. From now on, the only behaviour will be ``drawtype='box'``.
+
+Support for ``drawtype=line`` is removed altogether. As a
+result, the *lineprops* keyword argument to
+`~matplotlib.widgets.RectangleSelector` is also removed.
+
+To retain the behaviour of ``drawtype='none'``, use ``rectprops={'visible':
+False}`` to make the drawn `~matplotlib.patches.Rectangle` invisible.
+
+Cleaned up attributes and arguments are:
+
+- The ``active_handle`` attribute has been privatized and removed.
+- The ``drawtype`` attribute has been privatized and removed.
+- The ``eventpress`` attribute has been privatized and removed.
+- The ``eventrelease`` attribute has been privatized and removed.
+- The ``interactive`` attribute has been privatized and removed.
+- The *marker_props* argument is removed, use *handle_props* instead.
+- The *maxdist* argument is removed, use *grab_range* instead.
+- The *rectprops* argument is removed, use *props* instead.
+- The ``rectprops`` attribute has been privatized and removed.
+- The ``state`` attribute has been privatized and removed.
+- The ``to_draw`` attribute has been privatized and removed.
+
+PolygonSelector
+...............
+
+- The *line* attribute is removed. If you want to change the selector artist
+  properties, use the ``set_props`` or ``set_handle_props`` methods.
+- The *lineprops* argument is removed, use *props* instead.
+- The *markerprops* argument is removed, use *handle_props* instead.
+- The *maxdist* argument and attribute is removed, use *grab_range* instead.
+- The *vertex_select_radius* argument and attribute is removed, use
+  *grab_range* instead.
+
+SpanSelector
+............
+
+- The ``active_handle`` attribute has been privatized and removed.
+- The ``eventpress`` attribute has been privatized and removed.
+- The ``eventrelease`` attribute has been privatized and removed.
+- The *maxdist* argument and attribute is removed, use *grab_range* instead.
+- The ``pressv`` attribute has been privatized and removed.
+- The ``prev`` attribute has been privatized and removed.
+- The ``rect`` attribute has been privatized and removed.
+- The *rectprops* argument is removed, use *props* instead.
+- The ``rectprops`` attribute has been privatized and removed.
+- The *span_stays* argument is removed, use the *interactive* argument
+  instead.
+- The ``span_stays`` attribute has been privatized and removed.
+- The ``state`` attribute has been privatized and removed.
+
+LassoSelector
+.............
+
+- The *lineprops* argument is removed, use *props* instead.
+- The ``onpress`` and ``onrelease`` methods are removed. They are straight
+  aliases for ``press`` and ``release``.
```

Comment:
Removal of `TextBox.DIST_FROM_LEFT` is missing.

Reply:
FWIW this is all just copied and pasted from the 3.5 release notes and deprecated replaced with removed (more or less).

But I will of course edit accordingly.

Reply:
Looks like this was deprecated in bd9ee5e98616d96ab96fbef0d76d185bdf681cdd but we did not get a API change note with it.

Reply:
Should I update this to be deprecated in 3.7?

Reply:
It has been warning, but was not documented.  The conservative thing to do is put it back, document it removed in 3.7 and drop it in 3.8, but I'm inclined to be a bit YOLO here and just document that it is removed in 3.7 as this is a very niche feature...